bugsnag-android-core/src/androidTest/java/com/bugsnag/android/BreadcrumbFilterTest.kt

```diff
@@ -1,34 +1,46 @@
 package com.bugsnag.android
 
+import com.bugsnag.android.BreadcrumbType.MANUAL
+import org.junit.Assert.assertEquals
+
+import androidx.test.filters.SmallTest
 import com.bugsnag.android.BugsnagTestUtils.generateClient
-import com.bugsnag.android.BugsnagTestUtils.generateConfiguration
+
 import org.junit.After
-import org.junit.Assert.assertEquals
-import org.junit.Before
 import org.junit.Test
 
+@SmallTest
 class BreadcrumbFilterTest {
 
     private var client: Client? = null
 
-    @Before
-    fun setUp() {
-        val configuration = generateConfiguration()
-        configuration.enabledBreadcrumbTypes = setOf(BreadcrumbType.REQUEST)
-        client = generateClient(configuration)
-    }
-
     @After
     fun tearDown() {
-        client!!.close()
+        client?.close()
```

Comment:
This is an existing issue, but declaring the client as `lateinit var client: Client` would avoid having to use the safe operator at all.